CONRTIBUTING.md

```diff
@@ -61,6 +61,28 @@ __The following approach aims to reconcile these two competing demands:__
 --------
 
 
+### Library Standards
+
+The team has agreed to consistently use several third-party Python packages.
+This list will likely be fairly dynamic as we are always looking for things
+that will increase our productivity. We have a strong preference for tools that
+follow the [Principle of Least Astonishment](http://c2.com/cgi/wiki?PrincipleOfLeastAstonishment)
+(also on [Wikipedia](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)).
+
+The list:
+
+  - [arrow](https://pypi.python.org/pypi/arrow): Date/Time utilities
+  - [bottle](https://pypi.python.org/pypi/bottle): Web application micro-framework
+  - [cryptography](https://pypi.python.org/pypi/cryptography): cryptographic recipes and primitives
+  - [eventlet](https://pypi.python.org/pypi/eventlet): concurrent networking library
+  - [pymongo](https://pypi.python.org/pypi/pymongo): MongoDB driver
+  - [redis](https://pypi.python.org/pypi/redis): Redis client library
+  - [requests](https://pypi.python.org/pypi/requests): makes http requests virtually painless
+  - [tornado](https://pypi.python.org/pypi/tornado): concurrent networking library
```

Comment:
@samstav The only reason for choosing Tornado over another async package is that I introduced Tornado when implementing Patrick as our first Python 3 project. The only reason Tornado was used instead of our team standard of eventlet was because, at the time, eventlet wasn't working with bottle in Python 3.

@ziadsawalha has since mentioned that Tornado also supports bottle's reloader feature which is convenient during development.

I don't have a good answer here. I've thrown away several versions of this final paragraph. I will gladly consider recommendations and change the PR to match our decision.